doc/wg/core/notes/core-notes-2025-01-15.md

```diff
@@ -0,0 +1,109 @@
+# Tock Meeting Notes 2025-01-15
+
+## Attendees
+- Branden Ghena
+- Leon Schuermann
+- Pat Pannuto
+- Hudson Ayers
+- Johnathan Van Why
+- Kat Fox
+- Brad Campbell
+- Alexandru Radovici
+- Chris Frantz
+
+## Updates
+ * Leon: Alistair asked to add a TickV release tag to the Tock repo, which I added. That tag coincides with the Tock 2.2 release and marks TickV 2.0.0 release within Tock.
+ * Branden: After a hiatus for holidays and conferences, we did a walk-through of all the Tock networking interfaces, what their state is, what their goals are, and what the blockers are. That might be interesting for others to look at, so see the notes from that call.
+ * Branden: https://github.com/tock/tock/pull/4307
+ * Hudson: What was the status of 15.4 for the release?
+ * Leon: UDP remains broken, and has likely been broken for a few years now. 15.4 raw is still working.
+ * Hudson: If I set aside time, is there interest in UDP in 15.4? Or is that deprecated?
+ * Leon: I think either is good. It should probably be fixed or officially deprecated though. You might check in with Tyler about it
+ * Hudson: I'll put that on my list then
+
+
+## Syscall Drivers Testability
+ * https://github.com/tock/tock/issues/4303
+ * Alex: We need a testing framework for capsules and need to see what the capsule returns.
+ * Alex: We need unit tests for sure. A problem with them is we can simulate things but we can't extract the CommandReturn. The proper function for transforming it is within the kernel crate. So we can't use it without the kernel
+ * Leon: I can speak to why it's like that. For 2.0 we wanted CommandReturn to be entirely separate from SyscallReturn. Command returns come from commands regardless of system. And Syscall returns are how we send things to userspace. The two were supposed to stay separate, but with a way to transform one into the other.
```

Comment:
I think @lschuermann's point was: Command can only output a subset of syscall return variants (e.g. it shouldn't return pointers). CommandReturn intentionally hides that it wraps a SyscallReturn so it can guarantee that it only contains variants that Command can return.

Reply:
I edited history to pretend you had said that.